test/e2e/network/loadbalancer.go

```diff
@@ -1284,6 +968,14 @@ var _ = common.SIGDescribe("LoadBalancers ExternalTrafficPolicy: Local", framewo
 			framework.ExpectNoError(err)
 		})
 
+		// FIXME: figure out the actual expected semantics for
+		// "ExternalTrafficPolicy: Local" + "IPMode: Proxy".
+		// https://issues.k8s.io/123714
+		ingress := &svc.Status.LoadBalancer.Ingress[0]
+		if ingress.IP == "" || (ingress.IPMode != nil && *ingress.IPMode == v1.LoadBalancerIPModeProxy) {
```

Comment:
Line 1028 should only do the check if is not proxy ... in that case I dont think we should skip the test though

Reply:
Line 1278 hsa another FIXME related

Reply:
The NodePort test (at line 1028) isn't even using a LoadBalancer service. (Yeah, the test case is in the wrong file.) It's connecting from a pod to a nodeport, so that's just exercising kube-proxy behavior... which we know has to work because otherwise this test wouldn't pass in the gce tests! Not sure what's going on here.

The line 1278 FIXME doesn't affect our CI, but I believe that test case would fail in a cluster running Cilium, because it goes out of its way to preserve the source IP even when it "doesn't have to", https://docs.cilium.io/en/stable/network/kubernetes/kubeproxy-free/#client-source-ip-preservation